index.bs

```diff
@@ -186,8 +186,7 @@ below and in [[#index-defined-elsewhere]].
 
 : HTML
 :: The concepts of [=relevant settings object=], [=origin=],
-    [=opaque origin=], [=is a registrable domain suffix of or is equal to=], and the {{Navigator}} interface are
-    defined in [[!HTML52]].
+    [=opaque origin=], [=is a registrable domain suffix of or is equal to=].
```

Comment:
Fixed in https://github.com/w3c/webauthn/pull/384/commits/b1e87ca8f5feb2685dc9d47f033207e5972f86cc